.circleci/config.yml

```diff
@@ -155,6 +155,22 @@ jobs:
             just simulate-council
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
+  just_simulate_sep-op-006-0:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: just simulate sep/006-0-sc-guardian-updates
+          command: |
+            go install github.com/mikefarah/yq/v4@latest
+            just install
+            cd tasks/sep/006-0-sc-guardian-updates
```

Comment:
The CI failure is due to a path mismatch:
- On L164 and L168 in this file we use `sep/006-0-sc-guardian-updates`
- The actual path is `sep/006-0-sc-changes`

Reply:
Right, I had renamed the path. Thanks.